File: Data Structures & Algorithms/products-of-array-discluding-self/submission-2.cpp
class Solution {
public:


std::vector<int> productExceptSelf(const std::vector<int> &nums)
{

    std::vector<int> res;
    res.reserve(nums.size());

    for (int i = 0; i < nums.size(); i++)
    {
        int total = 1;
        for (int j = 0; j < nums.size(); j++)
        {

            if (i == j)
            {
                continue;
            }
            total *= nums[j];
        }

        res.push_back(total);
    }
    return res;
}


};

File: Data Structures & Algorithms/duplicate-integer/submission-2.java
class Solution {
    public boolean hasDuplicate(int[] nums) {
        
    Arrays.sort(nums);

    final int SIZE = nums.length;

    if(SIZE <= 1)
        return false;


    for(int i = 0 ; i < SIZE - 1 ; i++){

        if(nums[i] == nums[i + 1])
            return true;

    }
        

    return false;  

    }
    
}
